components-core/src/main/java/com/adyen/checkout/components/ui/Validation.kt

```diff
@@ -19,7 +17,7 @@ sealed class Validation {
     /**
      * Field is not valid.
      */
-    class Invalid(@StringRes val reason: Int) : Validation()
+    object Invalid : Validation()
```

Comment:
Why are we making this change?

Reply:
Hmm i see it's unused, but can we make use of it somehow instead of removing it?

Reply:
Yeah the idea was that the ViewModel could pass more descriptive error messages depending on why the validation failed, but I think this wasn't implemented yet.